test/e2e/policies.test-suite/topology-aware/c4pmem4/test05-guarantee-memory/code.var.sh

```diff
@@ -1,40 +1,152 @@
-CRI_RESMGR_OUTPUT="cat cri-resmgr.output.txt"
+CRI_RESMGR_OUTPUT="cat cri-resmgr.output.txt | tr -d '\0'"
+CRI_RESMGR_ROTATE="echo > cri-resmgr.output.txt"
 
-# pod0: 2 containers, both get their own socket and take > 50 % of their socket's mem
-CPU=200m MEM=5G CONTCOUNT=2 create guaranteed
+podno=0
+kubectl delete pod --all --now
+
+# account for being done with test for the current pod
+nextpod () {
+    podno=$((podno+1))
+}
+
+# print current pod name
+pod () {
+    echo pod$podno
+}
+
+# print current container name, by default for current pod
+container () {
+    local _p _c
+    case $# in
+        0) _p=${podno}; _c=0;;
+        1) _p=${podno}; _c=$1;;
+        2) _p=$1; _c=$2;;
+        *)
+            echo podWTFcHUHwhy${#}ARGS
```

Comment:
Change the error to something more readable(?)

Reply:
That's not an error message. It is a generated invalid container name. It will trigger
an assertion failure and it will show up in the related error message. We do not want
to bail out with a simple shell error. We want to keep going and run all the remaining
tests (`make -k` vs. `make` semantics). But we want to give a strong hint to the test
author that it is the test case code which is broken.

Okay, so maybe something like `podWTFcHUHwhy4ARGS` is not a good choice.
Changed it so now we'll produce something like

```
### Assertion failed for 'len(mems['pod0c1_INVALID_CALL_WITH_EXTRA_2_ARGS_xyzzy_foobar']) == 2'
```

Reply:
Looks better 😆